<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CM_NewDocumentDlg</class>
 <widget class="QDialog" name="CM_NewDocumentDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>好</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>标题:</string>
   </property>
   <property name="buddy">
    <cstring>editTitle</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="editTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listPreset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>详细设置</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>54</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>长度:</string>
    </property>
    <property name="buddy">
     <cstring>spinWidth</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinWidth">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> px</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2048</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>54</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>宽度:</string>
    </property>
    <property name="buddy">
     <cstring>spinHeight</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinHeight">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> px</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2048</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkLocks">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>锁定比例</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>像素长宽比:</string>
    </property>
    <property name="buddy">
     <cstring>comboRatio</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="comboRatio">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>方形像素</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>美式标准名片尺寸 (16:9)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>欧式标准名片尺寸 (16:10)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>窄式标准名片尺寸 (2:1)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>超窄标准名片尺寸 (9:4)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>折卡(折叠名片)标准尺寸 (18:19)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>长折卡(加长型折叠名片)标准尺寸 (13:5)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黄金矩形标准尺寸 (5:3)</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>背景:</string>
    </property>
    <property name="buddy">
     <cstring>comboBackgroundType</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBackgroundType">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>背景色</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>图片</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>透明</string>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="bg_color">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑颜色:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnColor">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bg_pic">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>文件位置:</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>7</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>图片位置:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboPicMode">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>拉伸</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>居中</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>平铺</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>缩放</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>预设:</string>
   </property>
  </widget>
  <widget class="QLabel" name="preview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>401</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
